### ISSUE TITLE
Shaka T-Shirt Raffle

### ISSUE THREAD

**Issue by joeyparrish** [google/shaka-player#2961]
**TL;DR: Fill out a form, maybe win a T-shirt!  Past PR contributors get 5x odds of winning!**

At the end of 2019, we made all these lovely T-shirts to give away at events in 2020.  And then all the events had to be cancelled!

So we have decided to end 2020 by giving away 20 Shaka T-shirts by mail to the GitHub communities.  Simply fill out the form linked below with your email address (used to contact winners) and GitHub username (used to grant increased odds to past contributors).  This information will not be used for any other purpose, nor shared with anyone else.

#### Details:

The raffle will close on December 31, 2020.  After that, 20 entrants will be randomly selected to receive a T-shirt and a Shaka sticker.  (See T-shirt designs, cuts, and sizes below.)

If you are selected as a winner, we will contact you via email to request your shipping address and your preferred cut & size of T-shirt.  T-shirts will be sent via the US Postal Service.  Your shipping address and T-shirt size will not be used for any other purpose.  We cannot make any guarantees about delivery times.

In order for us to comply with US export regulations, winners must be screened.  Residents of certain countries will be ineligible, as will certain individuals.  We apologize for any inconvenience this may cause you.  Screening will be conducted by Google before the announcement of the winners, and may take some time to complete.  We will announce winners in early 2021, once our internal screening process is complete.

For shipping purposes, non-US winners must act as the "importer of record", which means that you would be responsible for any duties/taxes on the shipment and/or whatever certification is required in your country of residence to receive a T-shirt and stickers.  If you are unwilling to do this, please do not enter the raffle.

#### Shirt Designs, Cuts, and Sizes:

![shaka-shirt-preview](https://user-images.githubusercontent.com/6843156/98052870-28448a80-1dec-11eb-96d2-8fd0747aaf22.png)

![shaka-styles-preview](https://user-images.githubusercontent.com/6843156/98052838-1cf15f00-1dec-11eb-9ad8-303db56159c0.png)

 - Unisex sizes:  Small, Medium, Large, XL, 2XL, 3XL
 - Women's sizes:  Small, Medium, Large, XL, 2XL
 - Tall sizes:  Large (LT), XLT, 2XLT

**Comment by joeyparrish** [comment]
The raffle is now closed.  Thanks to the 106 people who entered, 20 of whom were contributors to at least one Shaka project before the raffle began!

Now we will choose 20 winners at random.  Contributors' odds of winning are 5x, whether or not their PR was accepted.  The full code used for selecting winners can be found here: https://gist.github.com/joeyparrish/64acb427082853e502ee5389d1f92b58

Before we can announce the winners, they must be screened by Google to comply with export regulations.  Stay tuned!

**Comment by joeyparrish** [comment]
Emails have been sent to our winners.  Please check your spam filters!

**Comment by joeyparrish** [comment]
_**Congratulations to our raffle winners:**_
:tada: @sr1990, @arturparkhisenko, @hochhaus, @andrewnryan, @stansbridge, @sh0seo, @mishazawa, @geneticgenesis, @kumarashu123, @colinbarr, @rgc, @krzemienski, @thejohnbowers, @thijsl, @avelad, @sarge, @valotvince, @milosrasic, @colde, @gerardsoleca :tada:

In the form I sent asking winners for their addresses, I neglected to add an explicit field for name/github/email, and so I can't tell who did or didn't fill out the form.  But 1 of the 20 of you didn't.  :-)  So if you didn't fill out the form I sent, please do!  And if you didn't get it, please tag me and let me know so I can send it to you again.  In the meantime, I'll be putting 19 of the 20 shirts in the mail soon.

Thanks for playing, everyone!  And thanks to all of our wonderful contributors in all of our projects for helping to keep the Shaka family going.  :call_me_hand: :metal:

**Comment by joeyparrish** [comment]
Also, one more note: To the person who submitted an address of the form **"_X_ North _Y_ West"**, that's either a street address without a city, or coordinates in the Atlantic ocean.  Please submit the form again.  :grin:
